### PR TITLE
feat: sets the consistency level

### PR DESCRIPTION
**Pull request by shubhank-v**
### What changes are proposed in this PR?
---
* This PR sets the consistency level to LOCAL_QUORUM in CassandraConnectionManager.

### How was this tested?
---
* Will test in Airflow

### CODE REVIEW

**Review by vincentkoc — APPROVED** [review]

File: hip_data_tools/common.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
                          "most_specific_location"]
 
 LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
```
Comment (tusharm) [inline hip_data_tools/common.py:21]: can we hook this to a flag or an env?  just to avoid all consumers to start flooding cloudwatch..

**Review by vincentkoc — CHANGES_REQUESTED** [review]
See comment from @tusharm

File: hip_data_tools/common.py (hunk 1 above)
Comment (shubhank-v) [reply hip_data_tools/common.py]: toggling via env varirable

**Review by Rukesh-Kapuluru — APPROVED** [review]

**Review by Rukesh-Kapuluru — APPROVED** [review]

**Review by vincentkoc — APPROVED** [review]
Thanks @Rukesh-Kapuluru

**Pull request merged**